README.md

```diff
@@ -300,6 +300,14 @@ Past Carbon focused talks from the community:
     CppNorth 2022
 -   [Carbon Language: Syntax and trade-offs](https://www.youtube.com/watch?v=9Y2ivB8VaIs),
     Core C++ 2022
+-   [Carbon’s Successor Strategy: From C++ interop to memory safety (Slides)](https://chandlerc.blog/slides/2023-cppnow-carbon-strategy/index.html#/),
+    C++Now 2023
+-   [Definition-Checked Generics (Part 1) (Slides)](https://chandlerc.blog/slides/2023-cppnow-generics-1/#/),
+    C++Now 2023
+-   [Definition-Checked Generics (Part 2) (Slides)](https://chandlerc.blog/slides/2023-cppnow-generics-2/#/),
+    C++Now 2023
+-   [Modernizing Compiler Design for Carbon’s Toolchain (Slides)](https://chandlerc.blog/slides/2023-cppnow-compiler/index.html#/),
+    C++Now 2023
```

Comment:
```suggestion
-   C++Now 2023:
    -   Carbon’s Successor Strategy: From C++ interop to memory safety [(Slides)](https://chandlerc.blog/slides/2023-cppnow-carbon-strategy/index.html#/)
    -   Definition-Checked Generics: Part 1 [(Slides)](https://chandlerc.blog/slides/2023-cppnow-generics-1/#/) and Part 2 [(Slides)](https://chandlerc.blog/slides/2023-cppnow-generics-2/#/)
    -   Modernizing Compiler Design for Carbon’s Toolchain [(Slides)](https://chandlerc.blog/slides/2023-cppnow-compiler/index.html#/)
```

How does this format read to you? I'm wondering if it's worth it to switch a little so that we don't repeat as much. Also, maybe we should decide whether to link video/slides. But also, if you just want to merge as-is, I think that's reasonable too.

Reply:
I thought about that, I think it creates an asymmetry in the content that I don't think is justified (all talks have the same as importance). But maybe we could separate in 2022, and 2023? That way we see how recent those are more easily. Still some repetition with the event name I suppose, but I think that's acceptable?

Reply:
Good point for the two parts, merged. Added year too.
The reason for the slides is that only slides are available currently for C++Now. Otherwise video would be preferred (at least for consistency if anything).